tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -1297,6 +1297,39 @@ static struct device_info boards[] = {
 		.last_sysupgrade_partition = "file-system"
 	},
 
+	/** Firmware layout for the TL-WPA8630P v2 */
+	{
+		.id     = "TLWPA8630PV2",
+		.vendor = "",
+		.support_list =
+			"SupportList:\n"
+			"{product_name:TL-WPA8630P,product_ver:2.0.0,special_id:44450000}\n",
+		.support_trail = '\x00',
+		.soft_ver = NULL,
+		.partitions = {
+			{"factory-boot", 0x00000, 0x20000},
+			{"fs-uboot", 0x20000, 0x20000},
+			{"firmware", 0x40000, 0x5e0000},
+			{"partition-table", 0x620000, 0x02000},
+			{"extra-para", 0x632100, 0x01000},
```

Comment:
In DTS, you have info partition at 0x630000 with MAC address in 0x8. I wonder why that one is missing here.

Reply:
Damn it, I mixed up P and non-P version once again! I though I had that sorted out...